app/controllers/box_requests_controller.rb

```diff
@@ -9,7 +9,7 @@ def index
     if params[:filter_by].present?
       (filter_attr = params[:filter_by])
       @box_requests = @box_requests.public_send(filter_attr)
-      @box_requests.where(reviewed_by_id: params[:current_user_id]) if params[:current_user_id].present?
+      @box_requests = @box_requests.where(reviewed_by_id: params[:current_user_id]) if params[:current_user_id].present?
```

Comment:
Is it safe to make this stricter? For example, if there is supposed to be no way to get here without logging in, I'd be happy if this raises an exception when `params[:current_user_id]` is missing